src/index.spec.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const mock = require('mock-require');
+
+describe('Index', () => {
+    let index;
+
+    beforeAll(() => {
+        mock('path', {
+            join: function () {
+                return './src/helpers/mock-config.json';
```

Comment:
The `helpers` directory is specifically used by the [coverage reporter](https://github.com/blackbaud/skyux-builder-stache-search/pull/1/files#diff-ed46e9818c744eb24f78fcf29587d862R7) (confusing, I know). Mind adding the mock config to a `fixtures` directory, in the same location as the spec (`/src/fixtures`)?

Reply:
wat. :P I'll add it.